velox/common/memory/MappedMemory.h

```diff
@@ -147,20 +148,70 @@ class MappedMemory {
 
     // Returns the run number and the position within the run
     // corresponding to 'offset' from the start of 'this'.
-    void findRun(uint64_t offset, int32_t* index, int32_t* offsetInRun);
+    void findRun(
+        uint64_t offset,
+        int32_t* FOLLY_NONNULL index,
+        int32_t* FOLLY_NONNULL offsetInRun);
 
    private:
-    MappedMemory* mappedMemory_;
+    MappedMemory* FOLLY_NONNULL mappedMemory_;
     std::shared_ptr<MappedMemory> mappedMemoryPtr_;
     std::vector<PageRun> runs_;
     int32_t numPages_ = 0;
   };
 
+  // Represents a mmap'd run of contiguous pages that do not belong to
```

Comment:
MappedMemory appears to be a base class which is not specific to mmap/madvise-based implementation. Yet, it talks in terms of mmap and madvise. Would you clarify? 

What does numMapped() mean for malloc-based implementation?